dev/devicelab/bin/tasks/android_plugin_example_app_build_test.dart

```diff
@@ -10,12 +10,12 @@ import 'package:flutter_devicelab/framework/utils.dart';
 import 'package:path/path.dart' as path;
 
 final String gradlew = Platform.isWindows ? 'gradlew.bat' : 'gradlew';
-final String gradlewExecutable = Platform.isWindows ? '.\\$gradlew' : './$gradlew';
+final String gradlewExecutable =
+    Platform.isWindows ? '.\\$gradlew' : './$gradlew';
```

Comment:
nit: undo linebreak

Reply:
and elsewhere - it looks like dartfmt may have been applied where it shouldn't be.

Reply:
Done